ODIN_II/regression_test/benchmark/task/fpu/hard_logic/simulation_result.json

```diff
@@ -14,7 +14,7 @@
         "blif": "bgm.blif",
         "exit": 134,
         "errors": [
-            "[OUTPUT_BLIF] Could not resolve memory hard block fpu_mul~0 to a valid type."
+            "[OUTPUT_BLIF] Could not resolve memory hard block fpu_mul~6 to a valid type."
         ]
```

Comment:
The pin changed here which worries me

Reply:
I cant think of a good reason this would happen. I have a couple of theories, since I changed the read_blif slightly and also output slightly different which may cause the BLIF to encounter them in a different order

Reply:
https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/1398/files#diff-ab25514ea7cdfbd91f356b27ad8c7ceeR2588 this and this https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/1398/files#diff-f48accd8c411d18c3edaba9f7a21dc29R688 are my main thoughts

Reply:
@jeanlego Is there anything more you would like me to do on this?
I haven't been able to identify a specific reason why the count is different but I personally don't think this is an issue.